moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -73,52 +73,33 @@ int const THREAD_PRIORITY = 40;
 
 // Constructor for the class that handles servoing calculations
 ServoCalcs::ServoCalcs(const rclcpp::Node::SharedPtr& node,
-                       const std::shared_ptr<const moveit_servo::ServoParameters>& parameters,
-                       const planning_scene_monitor::PlanningSceneMonitorPtr& planning_scene_monitor)
+                       const planning_scene_monitor::PlanningSceneMonitorPtr& planning_scene_monitor,
+                       std::shared_ptr<const servo::ParamListener>& servo_param_listener)
   : node_(node)
-  , parameters_(parameters)
+  , servo_param_listener_(servo_param_listener)
+  , servo_params_(servo_param_listener_->get_params())
```

Comment:
If this is in the constructor, does it mean that none of the parameters can be dynamically updated at runtime?

EDIT: Ah, never mind, I see `updateParams()` now.